src/main/java/com/thebois/views/game/StructureView.java

```diff
@@ -26,19 +27,29 @@ public StructureView(final float tileSize) {
         this.tileSize = tileSize;
 
         houseTexture = TextureUtils.createSquareTexture(tileSize);
+        ceilingTexture = TextureUtils.createTriangleTexture(tileSize);
     }
 
     @Override
     public void draw(final Batch batch, final float offsetX, final float offsetY) {
         batch.setColor(houseColor);
         for (final IStructure structure : structures) {
+            // Draws Main house
             batch.setColor(houseColor);
             batch.draw(
                 houseTexture,
                 offsetX + structure.getPosition().getPosX() * tileSize,
                 offsetY + structure.getPosition().getPosY() * tileSize,
                 tileSize,
-                tileSize);
+                tileSize / 2);
+            // Draws roof
+            batch.setColor(Color.BLACK);
+            batch.draw(
+                ceilingTexture,
+                offsetX + structure.getPosition().getPosX() * tileSize,
+                offsetY + structure.getPosition().getPosY() * tileSize + tileSize / 2,
+                tileSize,
+                tileSize / 2);
```

Comment:
They are very similar but not exactly :P Maybe at least split into `draw(House)Base` and `drawRoof`